src/Components/ArticleCard.jsx

```diff
@@ -0,0 +1,25 @@
+export default function ArticleCard(props) {
+  const { article } = props;
```

Comment:
You could move { article } into the parameter instead of props on the line above to eliminate the need for this line

Reply:
done